src/source/Stun/StunUtils.c

```diff
@@ -0,0 +1,40 @@
+#define LOG_CLASS "STUNUtils"
+#include "../Include_i.h"
+#include "kvsstun/stun_data_types.h"
+
+STATUS convertStunErrorCode(StunResult_t stunResult)
```

Comment:
Can we add a unit test for this? Similar to https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/develop/tst/SdpApiTest.cpp#L62-L103

Reply:
Unit test in StunApiTest.c?

Reply:
Yes. In StunApiTest.

Reply:
Updated.